File: kframework/src/main/java/com/parallaxbacklayout/transform/ITransform.java
package com.parallaxbacklayout.transform;

import android.graphics.Canvas;
import android.view.View;

import com.parallaxbacklayout.widget.ParallaxBackLayout;


/**
 * ParallaxBackLayout
 *
 * @author An Zewei (anzewei88[at]gmail[dot]com)
 * @since ${VERSION}
 */

public interface ITransform {
    void transform(Canvas canvas, ParallaxBackLayout parallaxBackLayout, View child);
}

File: kframework/src/main/java/showmethe/github/kframework/parallaxbacklayout/transform/ITransform.kt
package showmethe.github.kframework.parallaxbacklayout.transform

import android.graphics.Canvas
import android.view.View

import showmethe.github.kframework.parallaxbacklayout.widget.ParallaxBackLayout


interface ITransform {
    fun transform(canvas: Canvas, parallaxBackLayout: ParallaxBackLayout, child: View)
}
